assets/src/components/d3/createHistogram.js

```diff
@@ -84,21 +122,41 @@ function createHistogram ({ data, width, height, domElement, xAxisLabel, yAxisLa
     svg.append('line')
       .attr(`transform`, `translate(0, ${aHeight - margin.bottom})`)
       .attr('x1', x(myGrade))
-      .attr('y1', -aHeight)
+      .attr('y1', -aHeight + 75)
       .attr('x2', x(myGrade))
       .attr('y2', 0)
       .attr('stroke', 'darkorange')
       .attr('stroke-width', '2')
     svg.append('text')
-      .attr('x', x(myGrade) - 110)
-      .attr('y', margin.top-5)
-      .text(`My Grade: ${roundToOneDecimal(myGrade)}%`)
+      .attr('x', myGrade < 5 ? x(myGrade) + 3 : x(myGrade) - 110)
+      .attr('y', margin.top + 15)
+      .text(`My Grade:${roundToOneDecimal(myGrade)}%`)
```

Comment:
Add a space so it's "My Grade: "? Also, sometimes the label covers part of the histogram. Would it be possible to add logic such that based on where the line is in the distribution, it would appear on the left or right of the line? Maybe that's a feature request for the future.

Reply:
have added spacing, But your idea for changing the `Mygrade` position based on bar height, let park it for now.